src/server/modules/user/__tests__/UserController.test.ts

```diff
@@ -22,6 +22,7 @@ const userAnnouncement = {
 
 describe('UserController', () => {
   const controller = new UserController(
+    // @ts-ignore - no need for private properties in UrlManagementService
```

Comment:
sorry, could you elaborate more on why `@ts-ignore` is needed here?

Reply:
tsc complains of the absence of the private properties that it expects to find in UrlManagementService. We cannot simply add private properties to the mock, since... I decided that removing interfaces is for the best.